examples/votchallenge/trackers.template.ini

```diff
@@ -0,0 +1,7 @@
+[MosseRust]
```

Comment:
Any preferences for name? The names in the results paper (https://www.votchallenge.net/vot2020/results.html are a mishmash of lowercase, CamelCase, Ugly_Case and ACRONYMS). I might be able to change it to mosse-tracker if you prefer.

Reply:
👍  looks good to me!